client/extensions/woocommerce/app/product-categories/form.js

```diff
@@ -54,10 +205,19 @@ class ProductCategoryForm extends Component {
 		return (
 			<div className={ classNames( 'product-categories__form', this.props.className ) }>
 				<Card>
-					<FormFieldSet>
-						<FormLabel htmlFor="name">{ translate( 'Category name' ) }</FormLabel>
-						<FormTextInput id="name" value={ category.name || '' } onChange={ this.setName } />
-					</FormFieldSet>
+					<div className="product-categories__form-info-fields">
+						<div className="product-categories__form-image-wrapper">{ this.renderImage() }</div>
+						<div className="product-categories__form-name-description">
+							<FormFieldSet>
+								<FormLabel htmlFor="name">{ translate( 'Category name' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **64** times:
`translate( 'Category Name' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/store-product-cat-form-fields)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).